Project.toml

```diff
@@ -5,11 +5,15 @@ version = "0.1.0"
 
 [deps]
 ArrayInterface = "4fba245c-0d91-5ea0-9b3e-6abc04ee57a9"
+IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
```

Comment:
Compats

Reply:
Still missing.

Reply:
yep sorry was in b2b meetings. adding `IterativeSolvers = "0.9.2"`.

Reply:
done